Justfile

```diff
@@ -55,7 +55,7 @@ mds *FLAGS:
     poetry run python -m ultravox.tools.mds_tool {{FLAGS}}
 
 gradio *FLAGS:
-    poetry run python -m ultravox.tools.gradio_demo {{FLAGS}}
+    poetry run gradio ultravox/tools/gradio_demo.py {{FLAGS}}
```

Comment:
How do you pass arguments with this change? It looks like the inference is not running on `mps` by default on Mac, and I got the following error when specifying the `--device` argument:
```
just gradio --device mps
poetry run gradio ultravox/tools/gradio_demo.py --device mps
Usage: gradio [OPTIONS] DEMO_PATH
Try 'gradio --help' for help.
╭─ Error ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ No such option: --device                                                                                                                                                                                                            │
╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
error: Recipe `gradio` failed on line 58 with exit code 2
```

Reply:
Hmm, it seems like it should work, per https://www.gradio.app/guides/developing-faster-with-reload-mode#python-ide-reload, but I'm getting the same error as you. I've asked on the Gradio Discord to try to figure out what we can do here, but in the short term I can try to make sure `mps` is used by default on Mac.

Reply:
Updated to make mps the default on Mac, as it seems to work pretty well now.